frontend/envs/env.dev

```diff
@@ -5,4 +5,6 @@ MSAL_USER_FLOW_NAME=B2C_1_susi_dev
 MSAL_USER_FLOW_AUTHORITY=https://rmiauthdev.b2clogin.com/rmiauthdev.onmicrosoft.com/B2C_1_susi_dev
 MSAL_AUTHORITY_DOMAIN=rmiauthdev.b2clogin.com
 MSAL_CLIENT_ID=218f47ee-11b1-459a-b914-b7fa6f107e7b
-MSAL_REDIRECT_URI=https://pacta.dev.rmi.siliconally.dev
+# See https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/initialization.md#redirecturi-considerations
+MSAL_REDIRECT_URI=https://pacta.dev.rmi.siliconally.dev/blank
```

Comment:
meganit - can we name this something clearer? `blank` has no context, and is more likely to be thought of as `new tab` etc. Very odd ms requires this - let's make the URL clear the purpose, since nothing else on the page will lol. `azure-ad-redirect-destination` ex

Reply:
This is a very reasonable and sensible comment, but I'm going to stick with `/blank` for a few reasons:

1. Pure laziness - I already submitted the TF PR and don't want to deploy both frontends + TF again
2. `/blank` is their recommendation, and I've seen similar things elsewhere. Not sure how strong of a convention it is, but getting a blank page at `/blank` seems reasonable.
3. Only frontend developers associate `new tab` and `blank` (i.e. `target=_blank`)
4. Nobody will ever actually see this, the only time it shows up is for 0.1seconds after signing in, in the Azure popup, just before it closes. The main site window will never show `/blank`